oonimkall/task_test.go

```diff
@@ -411,8 +411,8 @@ func TestIntegrationInterruptExampleWithInput(t *testing.T) {
 		"status.end",
 		"task_terminated",
 	}
-	if !reflect.DeepEqual(keys, expect) {
-		t.Fatalf("seen different keys than expected: %+v", keys)
+	if diff := cmp.Diff(expect, keys); diff != "" {
+		t.Fatalf(diff)
```

Comment:
This has been done to sync this test with another test that we changed to investigate failures